dev_guide/builds.adoc

```diff
@@ -385,6 +384,31 @@ sourceStrategy:
 You can also manage environment variables defined in the `*BuildConfig*` with the
 xref:../dev_guide/environment_variables.adoc#overview[`oc set env`] command.
 
+[[adding-secrets-to-source-strategy-build-configs]]
+=== Adding Secrets to Source Strategy Build Configurations
+
+Add a secret to your build configuration so that it can access a private
+repository.
+
+. Create a new {product-title} project.
```

Comment:
@ahardin-rh @jhadvig Why is a new project being created? Are the rest of the steps supposed to be in the new project? Is it used like a infra project so it's a house for the secrets?